modules/wildfly/src/main/modules/ogm/couchdb/module.xml

```diff
@@ -10,11 +10,12 @@
         <resource-root path="hibernate-ogm-couchdb-${project.version}.jar" />
     </resources>
     <dependencies>
-        <module name="org.hibernate" export="true" services="import" slot="${hibernate.ogm.module.slot}" />
+        <module name="org.hibernate" slot="${hibernate.ogm.module.slot}" />
         <module name="org.hibernate.hql" slot="${hibernate.hql.module.slot}" />
 
         <module name="javax.ws.rs.api" />
         <module name="javax.persistence.api" />
+        <module name="javax.transaction.api" />
```

Comment:
Because it would export it to all users (end users too), which is not needed. It's cleaner to have each module which actually needs it to state so explicitly: even if it results in a slightly larger total number or lines across all xml files, it results in smaller dependency graphs.

Generally speaking I've learned that every time you need to declare "export" there is something very wrong, I'm using it only as temporary workarounds for bugs, or circumvent limitations of modules which are not under our control.
For example I'm thinking we should not export OGM at all, but the ORM should import it as "optional".